frontend/simple/js/transitions.js

```diff
@@ -6,21 +6,25 @@ const Velocity = require('velocity-animate')
 
 // http://vuejs.org/guide/transitions.html#JavaScript-Transitions
 
-Vue.transition('fade', {
-  css: false,
-  enter: function (el, done) {
-    // https://github.com/oneuijs/You-Dont-Need-jQuery#css--style
-    el.style.opacity = 0
-    Velocity(el, { opacity: 1 }, 1000, done)
-  },
-  enterCancelled: function (el) {
-    Velocity(el, 'stop')
-  },
-  leave: function (el, done) {
-    // same as enter
-    Velocity(el, { opacity: 0 }, 1000, done)
-  },
-  leaveCancelled: function (el) {
-    Velocity(el, 'stop')
+Vue.component('fade', {
+  template: `<transition name="fade" v-on:enter="enter" v-on:enter-cancelled="enterCancelled" v-on:leave="leave"
+  v-on:leave-cancelled="leaveCancelled"></transition>
+`,
+  methods: {
+    enter: function (el, done) {
+      // https://github.com/oneuijs/You-Dont-Need-jQuery#css--style
+      el.style.opacity = 0
+      Velocity(el, { opacity: 1 }, 1000, done)
+    },
+    enterCancelled: function (el) {
+      Velocity(el, 'stop')
+    },
+    leave: function (el, done) {
+      // same as enter
+      Velocity(el, { opacity: 0 }, 1000, done)
+    },
+    leaveCancelled: function (el) {
+      Velocity(el, 'stop')
+    }
```

Comment:
Based on my cursory skimming of [the docs](https://vuejs.org/v2/guide/transitions.html), this doesn't seem to be the best/simplest, or even correct way to do this. Surely there's a simpler way?

FYI, this is used in `VueAssistant.vue`:

```js
    <component
      :is="view"
      :transition="transition"
      style="position: absolute; width: 100%"
    >
    </component>
```

I'm guessing there's a better way.